tutorial-create-custom-workflows/index.html

```diff
@@ -0,0 +1,173 @@
+
+<!doctype html>
+
+<html>
+<head>
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+  <meta name="theme-color" content="#4F7DC9">
+  <meta charset="UTF-8">
+  <title>Creating Custom Chaos Workflows</title>
+  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Source+Code+Pro:400|Roboto:400,300,400italic,500,700|Roboto+Mono">
+  <link rel="stylesheet" href="//fonts.googleapis.com/icon?family=Material+Icons">
+  <link rel="stylesheet" href="https://storage.googleapis.com/codelab-elements/codelab-elements.css">
+  <style>
+    .success {
+      color: #1e8e3e;
+    }
+    .error {
+      color: red;
+    }
+  </style>
+</head>
+<body>
+  <google-codelab-analytics gaid="UA-49880327-14"></google-codelab-analytics>
+  <google-codelab codelab-gaid=""
+                  id=""
+                  title="Creating Custom Chaos Workflows"
+                  environment="web"
+                  feedback-link="">
+    
+      <google-codelab-step label="Introduction" duration="1">
+        <h2 is-upgraded>Welcome</h2>
+<p>In this tutorial, you will learn how to create your own custom chaos workflows using Litmus. You will use ChaosHub and asset discovery to create your own custom Workflow.</p>
+<h2 is-upgraded>What you will learn</h2>
+<ul>
+<li>What is ChaosHub</li>
+<li>How to use Chaos Hub</li>
+<li>How to connect to a new Chaos Hub</li>
+<li>Create a custom workflow using Chaos Hub</li>
+</ul>
+<p>In this tutorial, you will inject a pod-delete fault against a sample microservices application called podtato-head and verify if the service continues to be available through the chaos duration.</p>
+<h2 is-upgraded>Pre-Requisites</h2>
+<ul>
+<li>Minikube 1.2.0 or later. Recommended sizing of at least: 2 vCPUs, 8GB memory, 10GB of disk space (for persistent volume creation)</li>
```

Comment:
Can be change this to be with respect to version of kubectl.
Like here https://litmuschaos.github.io/tutorials/tutorial-getting-started/index.html#0
Users can try this tutorial on other clusters also, so we should not mention Minikube.

Reply:
Fixed this.